### PR TITLE
use plugin-api wrapper; update imports for v6

### PR DESCRIPTION
**Pull request by wiggin77**
In order to use `mattermost-plugin-api` the imports have been updated to v6.

### CODE REVIEW

**Comment by wiggin77** [comment]
@mickmister can you add me as a collaborator?

**Comment by wiggin77** [comment]
@mickmister also can/should we rename this plugin + repo to something more generic?

**Comment by mickmister** [comment]
> also can/should we rename this plugin + repo to something more generic?

@wiggin77 Sure, how about `mattermost-plugin-user-channel-cleanup`? Doesn't really roll off the tongue well

File: server/plugin.go [hunk 1]
```diff
@@ -25,6 +26,8 @@ type Plugin struct {
 	plugin.MattermostPlugin
 	configurationLock sync.RWMutex
 	configuration     *configuration
+
+	papiClient *papi.Client
```
Comment (mickmister) [inline server/plugin.go:30]: In plugin projects I've worked on, we typically import the package as `pluginapi`, and store this as just `client`. 0/5, though I think consistency is good here

https://github.com/mattermost/mattermost-plugin-jira/blob/17665067caaf92bd8ffff6719edb6ee36dfdb534/server/plugin.go#L108
https://github.com/mattermost/mattermost-plugin-github/blob/11815f80ac05b110a5de479bbb7a9b1ff2995463/server/plugin/plugin.go#L71

**Review by mickmister — COMMENTED** [review]
So I originally had this as `mattermost-server/v5` to ensure it was compatible with the intended customer's system, though they have since upgraded so not an issue anymore. Great work on this :+1:

Just one comment for discussion

File: server/plugin.go (hunk 1 above)
Comment (wiggin77) [reply server/plugin.go]: Done

**Comment by wiggin77** [comment]
@mickmister I renamed papi and added a simple SQLStore.

**Review by mickmister — APPROVED** [review]
LGTM so far :+1: Awesome work @wiggin77

Let me know if you'd like this to be merged

**Pull request merged**